### DISCUSSION TITLE
q

### DISCUSSION THREAD

**Discussion by ahmadaliasghar** [vercel/next.js#68050] [App Router] [answered]

**Comment by knarrff** [comment, marked as answer]
Hi @ahmadaliasghar 

I assume from the existence of https://rasorce-new.vercel.app, which does look exactly like https://rasorce.co.uk/, that this was built using that framework. Which could mean that at some point in the past, google picked up that name from an old version of that website. If so, you can look through [this](https://developers.google.com/search/docs/appearance/site-names) and help Google choose a better name. That page has detailed examples of how to do that. My assumption is that once that has been done and Google happened to re-index your site, the info will be updated.

I hope this answered your question.
